hpOneView/resources/networking/interconnects.py

```diff
@@ -79,3 +79,14 @@ def get_subport_statistics(self, interconnect_id, port_name, subport_number):
         """
         uri = "/rest/interconnects/%s/statistics/%s/subport/%i" % (interconnect_id, port_name, subport_number)
         return self._client.get(uri)
+
+    def get(self, id):
+        """
+        Gets the Interconnect with the specified ID
+        Args:
+            id: ID of Interconnect
+
+        Returns: dict
+        """
+        uri = "%s/%s" % (self.URI, id)
```

Comment:
You don't need build the uri here. You can send only the ID and the ResourceClient does the job.